docs/export-custom-components.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Export Custom Questions to PDF | SurveyJS PDF Generator
+description: Learn how to export custom questions and third-party components to a PDF form.
+---
+
+# Export Custom Questions and Third-Party Components to PDF
+
+This help topic describes how to export custom questions that use [third-party components](https://surveyjs.io/survey-creator/documentation/customize-question-types/third-party-component-integration-react) to PDF. You can export custom questions as plain text, use a predefined renderer to export a custom question as an out-of-the-box question, or implement a custom PDF brick to customize the export as you like.
+
+## Render Custom Questions as Text
+
+SurveyJS PDF Generator exports custom questions as plain non-editable text that displays question values.
+
+![SurveyJS PDF Generator - Render custom questions as text](images/export-custom-question-as-text.png)
+
+Since this is the default behavior, you do not need to specifically configure it.
+
+## Use a Predefined Renderer
+
+If your custom question extends one of the out-of-the-box question types ([`QuestionTextModel`](https://surveyjs.io/form-library/documentation/api-reference/text-entry-question-model), [`QuestionCheckboxModel`](https://surveyjs.io/form-library/documentation/api-reference/checkbox-question-model), [`QuestionDropdownModel`](https://surveyjs.io/form-library/documentation/api-reference/dropdown-menu-model), etc.), it can use the renderer of the extended question type. For instance, the following code specifies to use a Text Input question renderer to export a custom `color-picker` question:

```

Comment:
```suggestion
If your custom question extends one of the out-of-the-box question types ([`QuestionTextModel`](https://surveyjs.io/form-library/documentation/api-reference/text-entry-question-model), [`QuestionCheckboxModel`](https://surveyjs.io/form-library/documentation/api-reference/checkbox-question-model), [`QuestionDropdownModel`](https://surveyjs.io/form-library/documentation/api-reference/dropdown-menu-model), etc.), it can use the renderer of the extended question type. For instance, the following code specifies the use of a Text Input question renderer to export a custom `color-picker` question:

```